db/re/item_db.conf

```diff
@@ -87001,6 +87258,29 @@ item_db: (
 	">
 },
 {
+	Id: 12888
+	AegisName: "Siege_Kit_Box"
+	Name: "Siege Kit Box"
+	Type: 2
+	Buy: 20
+	Weight: 100
+	Trade: {
+		nodrop: true
+		notrade: true
+		nocart: true
+		noselltonpc: true
+		nostorage: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		getrandgroupitem 12888,1;
+		getrandgroupitem 12888,1;
+		getrandgroupitem 12888,1;
```

Comment:
Nothing else in item_db.conf is using consts for getrandgroupitem. I didn't want to clutter up this commit by changing all the getrandgroupitem calls with const to make it uniform. That can be another commit.